test/autorest/generated/filegroup/internal/filegroup/models.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package filegroup
+
+import (
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"net/http"
+)
+
+type Error struct {
+	Message *string `json:"message,omitempty"`
```

Comment:
Shouldnt these have comments?

Reply:
Yes however they're missing from the swagger :(